modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -765,19 +765,32 @@ return L.view.extend({
 								if (nameval == null || protoval == null || nameval == '' || protoval == '')
 									return;
 
-								return m.save(function() {
-									var section_id = uci.add('network', 'interface', nameval);
+								return network.getProtocol(protoval).isCreateable(nameval).then(L.bind(function(checkval) {
+									if (checkval != null) {
+										ui.addNotification(null,
+												E('p', _('New interface for') + ' "' +
```

Comment:
Please change to:

`E('p', _('New interface for "%s" can not be created: %s').format(protoclass.getI18n(), checkval))`